scripts/setup.js

```diff
@@ -0,0 +1,93 @@
+
```

Comment:
No "use strict" here. Is that intentional?

Reply:
whoops! I've added it to all the files where it was missing